eval/eval.go

```diff
@@ -1120,18 +1128,31 @@ func (e *evalContext) evaluateBuiltinRotate(x *expr, repr *rotateExpr) *value {
 			e.execContext,
 		)
 		if err != nil {
+			diag := ast.ExprError(repr.syntax(), err.Error())
+			rotation := Rotation{
+				Path:   docPath,
+				Status: RotationFailed,
+				Diags:  []*syntax.Diagnostic{diag},
+			}
+			e.rotationResult.Rotations = append(e.rotationResult.Rotations, &rotation)
```

Comment:
```suggestion
			 e.rotationResult.Rotations = append(e.rotationResult.Rotations, &Rotation{
				Path:   docPath,
				Status: RotationFailed,
				Diags:  []*syntax.Diagnostic{diag},
			})
```

nit, just so it's the same as above ^
same for the following one